mappings/net/minecraft/entity/BlazeEntity.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_2661 net/minecraft/entity/BlazeEntity
+	METHOD method_10767 isBlazeOnFire ()Z
```

Comment:
you can probably remove the `blaze` here

Reply:
It's a duplicate. It's called by isOnFire and nowhere else.

Reply:
ah alright